game-app/game-core/src/main/java/tools/map/making/ui/xml/XmlUtilitiesPanel.java

```diff
@@ -1,30 +1,25 @@
 package tools.map.making.ui.xml;
 
-import javax.swing.Box;
-import javax.swing.BoxLayout;
+import java.util.List;
```

Comment:
note: if we skip updating this file, then we can simplify our builer API. 
The comments on the builder API assume we will just revert this last file (arguably we will want to redo how teh XMLuitility panels goes. so it's good for it to be independent of shared code)